core/src/main/java/com/amplifyframework/api/graphql/GraphQLRequest.java

```diff
@@ -69,6 +71,22 @@ public GraphQLRequest(
         this.variablesSerializer = variablesSerializer;
     }
 
+    /**
+     * Returns the query document string.
+     * @return the query document string
+     */
+    public String getDocument() {
+        return document;
+    }
+
+    /**
+     * Returns the immutable map of request variables.
+     * @return the map of variables
+     */
+    public Map<String, Object> getVariables() {
```

Comment:
Ah heck. I was gonna say, "remember to add this to the `hashCode()`/`equals()`/`toString()`!" But it looks like we don't even have those in this class. Which, given how prominent it is in the API signatures, is bad news.

Reply:
I removed `getVariables` and `getDocument` again after I realized that it isn't needed anymore. but I agree it still needs those methods you brought up.